src/System.ComponentModel.Annotations/tests/System/ComponentModel/DataAnnotations/RangeAttributeTests.cs

```diff
@@ -586,23 +587,25 @@ public static void DotDecimalExtremaAndInvalidValuesInvariantParse(Type type, st
         [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "2648 not fixed on NetFX")]
         public static void DotDecimalExtremaAndInvalidValuesInvariantConvert(Type type, string min, string max, string value)
         {
-            RemoteExecutor.Invoke((t, m1, m2, v) =>
+            RemoteExecutorForUap.Invoke((t, m1, m2, v) =>
             {
-                Thread.CurrentThread.CurrentCulture = CultureInfo.GetCultureInfoByIetfLanguageTag("en-US");
-
-                Assert.False(
-                    new RangeAttribute(Type.GetType(t), m1, m2)
+                using (new ThreadCultureChange("en-US"))
+                {
+                    Assert.False(
+                        new RangeAttribute(Type.GetType(t), m1, m2)
+                        {
+                            ConvertValueInInvariantCulture = true
+                        }.IsValid(v));
+                }
+
+                using (new ThreadCultureChange("fr-FR"))
+                {
+                    var range = new RangeAttribute(Type.GetType(t), m1, m2)
```

Comment:
Is there an auto-fixer that will put in 'var' where our style allows it? (And enforce it?)

Reply:
There's one built into the IDE, but it often gets it wrong, in both directions.